### DISCUSSION TITLE
Display a position in CLI

### DISCUSSION THREAD

**Discussion by artemmiyy** [LeelaChessZero/lc0#1991] [General]
Hey everyone. Is there a way to display a position in lc0's command line interface (analog of `d` in SF)?

**Comment by mooskagh** [comment]
There is a `fen` command, but nothing that draws a board.

**Reply by artemmiyy** [reply]
Do you think it'd be a worthwhile addition to lc0? If so, I can try to develop it. Please let me know what you think.

**Reply by mooskagh** [reply]
I don't think there's a need for it. UCI interface is really meant to be used by robots, not humans, and in cases we need to vizualize the position, `fen` command plus services like `http://lc0.org/fen/<fen_line>` (e.g. https://lc0.org/fen/rnbqkbnr/rnbppbnr/ppp2ppp/8/8/PPP2PPP/RNBPPBNR/RNBQKBNR) is enough.
